torch/lib/TH/generic/THTensorLapack.c

```diff
@@ -938,9 +938,12 @@ void THTensor_(ormqr)(THTensor *ra_, THTensor *a, THTensor *tau, THTensor *c, co
   THTensor_(free)(work);
 }
 
-void THTensor_(btrifact)(THTensor *ra_, THIntTensor *rpivots_, THIntTensor *rinfo_, THTensor *a)
+void THTensor_(btrifact)(THTensor *ra_, THIntTensor *rpivots_, THIntTensor *rinfo_, int pivot, THTensor *a)
 {
   THArgCheck(THTensor_(nDimension)(a) == 3, 1, "expected 3D tensor, got %dD", THTensor_(nDimension)(a));
+  if (!pivot) {
+    THError("btrifact is only implemented with pivoting on the CPU.");
```

Comment:
Ah, I think I misunderstood the meaning of the sentence then. Maybe it would be better to say something like
```
btrifact without pivoting is not implemented on the CPU
```
?